openvdb/version.h

```diff
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////////
 //
-// Copyright (c) 2012-2018 DreamWorks Animation LLC
+// Copyright (c) 2012-2019 DreamWorks Animation LLC
```

Comment:
@pcucka - I know in previous years you've done a bulk update of these copyright version numbers when the new year rolls around. Is the idea in just changing this one because the copyright should always be up-to-date for any files edited but it's not worth doing a bulk update because we're considering a move to SPDX soon?

Reply:
That was my thinking, yes.  Hopefully we can make a final decision soon on whether to remove the copyright dates, but maybe in the meantime I should just submit a PR with a bulk update.